scripts/simplify_network.py

```diff
@@ -620,6 +793,20 @@ def consense(x):
 
     update_p_nom_max(n)
 
+    p_threshold_drop_isolated = max(
+        0.0, cluster_config.get("p_threshold_drop_isolated", 0.0)
+    )
+    p_threshold_merge_isolated = cluster_config.get("p_threshold_merge_isolated", 0.0)
```

Comment:
In this case, let's have False as adefault: that means this merge is not executed.